src/Toolbox.php

```diff
@@ -311,9 +312,12 @@ private static function log($level = LogLevel::WARNING, $args = null)
         $tps = microtime(true);
 
         try {
+            $msg = self::cleanPaths($msg);
```

Comment:
I think it is not required. The `$PHPLOGGER` uses the `ErrorLogLineFormatter/AbstractLogLineFormatter` that already handle this clean.

```suggestion
```

Reply:
path cleaning is made in the LineFormatter attached to the ErrorLogHandler.
At the moment, the error handler is pushed in the logger. 
But **it may not be the case in the future**.
By doing it, we avoid a future regression.
Cleaning both times has no downside.

So you're right, **at the moment, with the current configuration** cleaning happens 2 times.
But it's future-proof to keep this filtering.

Reply:
OK.